pkg/sqlmigrations/migrations.go

```diff
@@ -1239,8 +1249,9 @@ func createNewSystemNamespaceDescriptor(ctx context.Context, r runner) error {
 		b := txn.NewBatch()
 
 		// Retrieve the existing namespace table's descriptor and change its name to
-		// "namespace_deprecated". This prevents name collisions with the new
-		// namespace table, for instance during backup.
+		// "namespace". This corrects the behavior of this migration as it existed
+		// in 20.1 betas. The old namespace table cannot be edited without breaking
+		// explicit selects from system.namespace in 19.2.
```

Comment:
Shouldn't we get rid of this part of the migration -- all it is doing is writing the same name back to the deprecated namespace table

Reply:
+1